extensions/amp-bind/amp-bind.md

```diff
@@ -991,7 +991,7 @@ Change the `width` and `height` using the `[width]` and `[height]` attributes.
 
 **Accessibility states and properties**
 
-Use to dynamically update information available to assistive technologies, such as screen readers. All `[aria-*]` and `[data-*]` are bindable. See the [full list here](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties).
+Use to dynamically update information available to assistive technologies, such as screen readers. [All `[aria-*]`](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties) and only [specific `[data-*]`](#amp-component-specific-attributes) attributes are bindable.
```

Comment:
I'm still confused on why we are mentioning `[data-*]` attributes at all within this section. My impression is that this section only applies to accessibility. Do the `data` attributes relate to accessibility in any way?

Reply:
Sleeping a night over this, I think you are right. I'll just remove the `[data-*]` part.

Reply:
The [aria docs](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties) do not mention [data-] attributes at all..